### PR TITLE
Remove enterprise requirement for sign language (BL-9877)

### PR DESCRIPTION
**Pull request by StephenMcConnel**
This change is [Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4420)

### CODE REVIEW

**Comment by hatton** [comment]
Seems like this is doing more than removing the enterprise requirement? The diff is hard to follow, so I could be wrong?

**Review by StephenMcConnel — COMMENTED** [review]
The problem is that removing the two outer elements that imposed enterprise results in re-indenting  a large section of jsx, which makes the diff problematic since re-indenting also causes some lines to collapse during the ensuing reformatting.
I removed every mention of enterprise in the tsx file, which changed a number of method signatures, and contents of some boolean expressions.

> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/4420#-:-MZngyHF96OLvY3JLMle:b-u68hd6)* status: 0 of 2 files reviewed, all discussions resolved
